linera-views/src/common.rs

```diff
@@ -209,43 +209,58 @@ fn lower_bound_test1_the_lower_bound() {
     set.insert(vec![24]);
     set.insert(vec![40]);
 
-    let mut lower_bound = GreatestLowerBoundIterator::new(0, set.iter());
-    assert_eq!(lower_bound.get_lower_bound(&[3]), None);
-    assert_eq!(lower_bound.get_lower_bound(&[15]), Some(vec!(10)).as_ref());
-    assert_eq!(lower_bound.get_lower_bound(&[17]), Some(vec!(10)).as_ref());
-    assert_eq!(lower_bound.get_lower_bound(&[25]), Some(vec!(24)).as_ref());
-    assert_eq!(lower_bound.get_lower_bound(&[27]), Some(vec!(24)).as_ref());
-    assert_eq!(lower_bound.get_lower_bound(&[42]), Some(vec!(40)).as_ref());
+    let mut suffix_closed_set = SuffixClosedSet::new(0, set.iter());
+    assert_eq!(suffix_closed_set.get_lower_bound(&[3]), None);
+    assert_eq!(
+        suffix_closed_set.get_lower_bound(&[15]),
+        Some(vec!(10)).as_ref()
```

Comment:
I think `vec![10]` is more common than `vec!(10)`.

Reply:
Ok.